templates/cypress.config.js

```diff
@@ -10,3 +10,24 @@ module.exports = {
 		specPattern: 'test/integration/**/**/*.test.ts',
 	},
 };
+
+if (process.env.WEB3_SYSTEM_TEST_CLIENT === 'firefox') {
+	const port = parseInt(String(Math.random() * 10000 + 10000));
```

Comment:
Is there a reason we can't just hardcode a port?

Reply:
Yes, it was a reason. I really do not remember why I need a random port. Looks like we need it for a parallel run